<launch>
    <node  launch-prefix="gdb -ex run --args" name="msf_pose_pressure_sensor" pkg="msf_updates" type="pose_pressure_sensor" clear_params="true" output="screen">
            <remap from="msf_core/imu_state_input" to="/odroidx/fcu/imu" />
            <remap from="msf_updates/pose_with_covariance_input" to="/odroidx/down/vslam/pose" />
            <remap from="msf_updates/pressure_input" to="/odroidx/fcu/imu_custom" />

            <rosparam file="$(find msf_updates)/pose_pressure_sensor_fix.yaml"/>
    </node>
</launch>
